doc/v2/ocpp_2x_status.md

```diff
@@ -2590,35 +2590,35 @@ This document contains the status of which OCPP 2.0.1 and OCPP2.1 numbered funct
 | ID        | Status                                          | Remark |
 | --------- | ----------------------------------------------- | ------ |
 |           | AllowedEnergyTransfer / RequestedEnergyTransfer |        |
-| Q01.FR.01 |                                                 |        |
-| Q01.FR.02 |                                                 |        |
-| Q01.FR.03 |                                                 |        |
-| Q01.FR.04 |                                                 |        |
-| Q01.FR.05 |                                                 |        |
-| Q01.FR.06 |                                                 |        |
-| Q01.FR.07 |                                                 |        |
-| Q01.FR.08 |                                                 |        |
-| Q01.FR.09 |                                                 |        |
-|           | Device model                                    |        |
-| Q01.FR.30 |                                                 |        |
-| Q01.FR.31 |                                                 |        |
-| Q01.FR.32 |                                                 |        |
+| Q01.FR.01 |   ✅ / 🌐                                       |        |
+| Q01.FR.02 |   ⛽️                                            | This is handled in OCPP2 module in EVerest. |
+| Q01.FR.03 |   ✅ / ⛽️                                       | Information for the message is filled in the OCPP2 module and libocpp sends the message. |
+| Q01.FR.04 |   🌐                                            |        |
+| Q01.FR.05 |   ❎                                            | In principle it is a charging station requirement, but should never occur / is not really possible to renegotiate at this time.       |
```

Comment:
Shouldnt we keep this open since its just not supported by libocpp at the moment?
```suggestion
| Q01.FR.05 |                                              | In principle it is a charging station requirement, but should never occur / is not really possible to renegotiate at this time.       |
```